src/Entity/Type/RefundEnumType.php

```diff
@@ -30,31 +32,36 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
         return 'VARCHAR(256)';
     }
 
-    public function convertToPHPValue($value, AbstractPlatform $platform): RefundType
+    public function convertToPHPValue($value, AbstractPlatform $platform): RefundTypeInterface
```

Comment:
```suggestion
    public function convertToPhpValue($value, AbstractPlatform $platform): RefundTypeInterface
```
this way is more camelCase :)
like `getId` instead `getID`

Reply:
I cannot change the name of this method, it is defined in class below